src/main/java/io/appform/dropwizard/sharding/observers/internal/TerminalTransactionObserver.java

```diff
@@ -0,0 +1,20 @@
+package io.appform.dropwizard.sharding.observers.internal;
+
+import io.appform.dropwizard.sharding.execution.TransactionExecutionContext;
+import io.appform.dropwizard.sharding.observers.TransactionObserver;
+
+import java.util.function.Supplier;
+
+/**
+ *
+ */
+public class TerminalTransactionObserver extends TransactionObserver {
```

Comment:
What does this observer signify ?

Reply:
This basically marks the end point of the observer chain. Not strictly necessary, but decided to have this in place so as to have a clean marker.